data/human/news.txt

```diff
@@ -219,6 +219,7 @@ news "street vendor"
 			"Want some budget caviar? What's caviar? It's horse beef, duh. Why do you think that people who ride horses are called cavalry... Fish eggs?"
 			"Here, have a sample of my famous lemonsausage. What's a lemonsausage? It's, erm... It's a blend of lemon and pig meat! Trust me! No, don't go away!"
 			"I wish I had enough money to ask captains roaming around the spaceport for transport to somewhere nicer."
+			"Sell a man a fish and you can sell him fish every day; teach a man to fish and you lose your customer for the rest of your lifetime."
```

Comment:
```suggestion
			"Sell a man a fish and you can sell him fish every day; teach a man to fish and you lose your customer."
```
I think the amusing aspect here hits better without the extra wording at the end. Up to you.